imagetest/testworkflow_test.go

```diff
@@ -16,8 +16,222 @@ package imagetest
 
 import (
 	"testing"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
 )
 
+func TestAddStartStep(t *testing.T) {
+	twf, err := NewTestWorkflow("name", "image")
+	if err != nil {
+		t.Errorf("failed to create test workflow: %v", err)
+	}
+	if twf.wf == nil {
+		t.Error("test workflow is malformed")
+	}
+	step, err := twf.addStartStep("stepname", "vmname")
+	if err != nil {
+		t.Errorf("failed to add start step to test workflow: %v", err)
+	}
+	if step.StartInstances == nil {
+		t.Error("StartInstances step is missing")
+	}
+	if len(step.StartInstances.Instances) != 1 {
+		t.Error("StartInstances step is malformed")
+	}
+	if step.StartInstances.Instances[0] != "vmname" {
+		t.Error("StartInstances step is malformed")
+	}
+	stepFromWF, ok := twf.wf.Steps["start-stepname"]
+	if !ok || step != stepFromWF {
```

Comment:
You can join these into one line:

`if stepFromWF, ok := twf.wf.Steps["start-stepname"]; !ok || step != stepFromWF {`

Reply:
(Same for the final check of the other newly-added tests too)

Reply:
ah yes, this originally did more with stepFromWF